.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -29,4 +29,15 @@ Unless otherwise noted above, this PR will be considered ready for review when a
 - [ ] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
 - [ ] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
 - [ ] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
-- [ ] All figures follow the [Manubot figure instructions](https://github.com/jaclyn-taroni/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
```

Comment:
Review me

Reply:
FWIW the link in the line below for the Manubot spellchecker won't resolve until this hits master. I wasn't sure of a better way to handle that though.